### DISCUSSION TITLE
Runners doesn't changes PHP version

### DISCUSSION THREAD

**Discussion by rogersferreira** [domcloud/domcloud#35] [Q&A]
I've tried to change my runner to something like:

features:
- php latest

features:
- php 8.1

features:
- php8.1

features:
- 'php latest'

I can't change the PHP version of my instances. 
Can anyone help me? Thanks in advance.

**Comment by willnode** [comment]
My bad. PHP 8.1 wasn't installed in NYC server. Should have been fixed now. Try again.

**Comment by rogersferreira** [comment]
Thank you for your time and good work, Wildan!
However, the problem now is another. Can you help me with that?

![image](https://user-images.githubusercontent.com/125833898/220622577-1951a605-1428-4413-85ab-56e80a7f66b2.png)

The runner:
![image](https://user-images.githubusercontent.com/125833898/220622854-a431b901-1b52-4911-8e9a-81a8d439ea57.png)

**Reply by willnode** [reply]
current latest PHP is 8.2... It wasn't installed. I've fixed that. Please try again.

**Reply by rogersferreira** [reply]
It works!!! Awesome!

Thank you very much Wildan!
I'm a teacher at a public school with techonology classes. All my students will benefit for your work! Thank you!
This fix is working in any server location or it will work only for New York?

**Reply by willnode** [reply]
On all server too
